### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by StephenGill**
Update privacy policy to reflect changes following new registrations being stopped: https://trello.com/c/nRqD3g2M/244-re-publish-wave-1-privacy-policy

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -106,57 +106,73 @@ en:
...
+        <li>name</li>
+        <li>address</li>
+        <li>date of birth</li>
+        <li>email address</li>
+        <li>phone numbers</li>
+        <li>NHS number</li>
+        <li>whether the person needs help getting food and essential supplies</li>
+        <li>whether the person needs help meeting their basic care needs</li>
+        <li>whether the person has special dietary requirements</li>
+        <li>Nwhether there’s someone in the person’s household who can carry a box of supplies inside</li>
```
Comment (JonathanHallam) [inline config/locales/en.yml]: don't think this N is meant to be there

Comment (StephenGill) [reply config/locales/en.yml]: Sorry, yes - it's a typo. Is it easier for you to fix it, or would you rather I did it?

Comment (JonathanHallam) [reply config/locales/en.yml]: I'll fix it now. There also seems to be something that's caused this not to work but I'm not sure what it is yet

**Review by issyl0 — APPROVED** [review]
Looks like some HTML in some YAML.

**Pull request merged**